xml/System/DateTime.xml

```diff
@@ -7080,7 +7080,7 @@ This member is an explicit interface member implementation. It can be used only
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- A single tick represents one hundred nanoseconds or one ten-millionth of a second. There are 10,000 ticks in a millisecond, or 10 million ticks in a second.  
+ A single tick represents one hundred nanoseconds or one ten-millionth of a second. There are 10,000 ticks in a millisecond (see <see cref="F:System.TimeSpan.TicksPerMillisecond" />) and 10 million ticks in a second.  
```

Comment:
```suggestion
 A single tick represents one hundred nanoseconds or one ten-millionth of a second. There are 10,000 ticks in a millisecond (see <xref:System.TimeSpan.TicksPerMillisecond>) and 10 million ticks in a second.  
```

Oops, I suggested the XML format for the link instead of the markdown format. Sorry! Here's the [preview link](System.TimeSpan.TicksPerMillisecond) for internal viewing.